api/infrastructure/handler/order/updateorder.go

```diff
@@ -0,0 +1,73 @@
+package order
+
+import (
+	"context"
+	"errors"
+	"net/http"
+
+	"github.com/danielgtaylor/huma/v2"
+	"shanepee.com/api/domain"
+	"shanepee.com/api/infrastructure/handler"
+	"shanepee.com/api/service"
+)
+
+type OrderUpdateBody struct {
+	TrackingNumber  *string             `json:"tracking_number,omitempty" example:"TH1234567890"`
+	DeliveryService *string             `json:"delivery_service,omitempty" example:"Kerry Express"`
+	Status          *domain.OrderStatus `json:"status" gorm:"not null" enum:"PREPARING,DELIVERING,COMPLETED" example:"pending"`
```

Comment:
we move off null laew kub
```suggestion
	Status          *domain.OrderStatus `json:"status,omitempty" gorm:"not null" enum:"PREPARING,DELIVERING,COMPLETED" example:"pending"`
```